hokusai/config.yml

```diff
@@ -1,4 +1,5 @@
 ---
-aws-account-id: '585031190124'
+aws-account-id: "585031190124"
 aws-ecr-region: us-east-1
 project-name: force
+post-deploy: 'title="Force was deployed"; text="force was deployed"; echo -e "_e{${#title},${#text}}:$title|$text|#shell,bash" > /dev/udp/datadog/8125'
```

Comment:
One glorious line! 😆 I wonder if it would be more clear without the intermediate bash variables, but if it works 🤷‍♂️

Reply:
Q: does this get the `service:force` tag automatically as well? How?